packages/pluginsdk-peerdeps/sync-save.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+/* @ts-check */
+/* eslint-disable no-console */
```

Comment:
This script synchronizes the contents of the scaffolded plugin's `package.json` and re-writes the `peerDependencies` and `devPeerDependencies` of the `@spinnaker/pluginsdk-peerdeps` package.